src/LiquidationPairFactory.sol

```diff
@@ -21,11 +21,15 @@ contract LiquidationPairFactory {
     /* ============ Variables ============ */
     LiquidationPair[] public allPairs;
```

Comment:
dont forget to add an array length accessor!

Reply:
Done: https://github.com/pooltogether/v5-liquidator/pull/2/commits/29759b223bba1cd49a6a2c7469683cd6e11b523d